Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs

```diff
@@ -63,7 +77,7 @@ void IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
             TryUseFood(eventArgs.User, eventArgs.Target);
         }
 
-        private bool TryUseFood(IEntity user, IEntity target)
+        internal bool TryUseFood(IEntity user, IEntity target, UtensilComponent utensilUsed = null)
```

Comment:
At this point you could just make it public, since `Content.Server` isn't accessible from anywhere else.